pyproject.toml

```diff
@@ -69,6 +70,17 @@ style = "pep440"
 [tool.black]
 line-length = 127
 
+[tool.isort]
```

Comment:
isort has a profile option that we should set to black https://pycqa.github.io/isort/docs/configuration/black_compatibility.html